scripts/rollup/mangle.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+// these are added as they are handled by rollup in some cases
+const propertyMangleWhitelist = [
+  // React
```

Comment:
It's not going to happen yet, I just added support to the tooling to be used when we get around to it. We need to change some things on master for this stuff to work anyway.